core/templates/components/entity-creation-services/topic-creation.service.ts

```diff
@@ -16,19 +16,35 @@
  * @fileoverview Modal and functionality for the create topic button.
  */
 
+require(
+  'components/forms/custom-forms-directives/select2-dropdown.directive.ts');
+require(
+  'components/forms/custom-forms-directives/thumbnail-uploader.directive.ts');
+require('domain/topic/topic-update.service.ts');
 require('domain/utilities/url-interpolation.service.ts');
 require('domain/topic/topic-creation-backend-api.service.ts');
+require('pages/topic-editor-page/services/topic-editor-state.service.ts');
 require('services/alerts.service.ts');
+require('services/image-upload-helper.service.ts');
+
+// TODO(#9186): Change variable name to 'constants' once this file
+// is migrated to Angular.
+const topicConstants2 = require('constants.ts');
```

Comment:
Why "2"? That's not a suitable name because it doesn't convey meaning...

Reply:
Not needed anymore, fixed.